src/batch.rs

```diff
@@ -68,7 +69,7 @@ impl Batch {
                 meta.size
                     .render(colors, max_size_value_length, max_size_unit_length),
                 ANSIString::from("  "),
-                meta.date.render(colors),
+                meta.date.render(colors, max_date_length, flags.date),
```

Comment:
You should give the Flags struct directly instead of a bool. It would avoid to change the `Render()` signature each time need an another flag (Spoiler: it's coming soon).